.github/workflows/golangci-lint.yml

```diff
@@ -11,8 +11,12 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.16.15
       - uses: actions/checkout@v2
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v2
         with:
           version: latest
+          skip-go-installation: true
```

Comment:
I think this is redundant. See https://github.com/golangci/golangci-lint-action#compatibility

Reply:
ah I see

Reply:
turns out it looks like we need this